font-codegen/src/parsing.rs

```diff
@@ -567,9 +573,27 @@ impl Parse for RawEnum {
         let name = input.parse::<syn::Ident>()?;
         let content;
         let _ = braced!(content in input);
-        let variants = Punctuated::<RawVariant, Token![,]>::parse_terminated(&content)?
+        let variants: Vec<_> = Punctuated::<RawVariant, Token![,]>::parse_terminated(&content)?
             .into_iter()
             .collect();
+
+        let defaults: Vec<_> = variants
+            .iter()
+            .filter_map(|v| {
+                if v.attrs.default {
+                    Some(v.name.clone())
+                } else {
+                    None
+                }
+            })
+            .collect();
+        if defaults.len() > 1 {
+            return Err(logged_syn_error(
+                name.span(),
+                format!("multiple defaults {defaults:?}"),
+            ));
+        }
```

Comment:
we have a pattern where we have a separate pass, after parsing, where we check that attributes make sense; this is the `sanity_check` method.

So what I would do is to just parse this naively here, and then add a `sanity_check` method to `RawEnum`, which checks that there's at most one item marked default.

(also you'll need to update `Item::sanity_check` to forward the call to the type; currently it doesn't, since we don't currently implement this for raw enums)

Reply:
I've been away from codegen too long, forgot ol' sanity check. Good catch, will do.

Reply:
Fixed.